src/main/java/tr/com/orioninc/laborant/exception/AlreadyExists.java

```diff
@@ -0,0 +1,7 @@
+package tr.com.orioninc.laborant.exception;
+
+public class AlreadyExists extends RuntimeException {
```

Comment:
Could you please rename all exceptions that extends a base Exception class?
They'll need "Exception" suffix at the end of their names, i.g. AlreadyExists to AlreadyExistsException